Get infernalis to build in openSUSE Build Service

smithfarm opened the pull request:
http://tracker.ceph.com/issues/13324 Fixes: #13324
http://tracker.ceph.com/issues/13346 Fixes: #13346
http://tracker.ceph.com/issues/13351 Fixes: #13351
http://tracker.ceph.com/issues/13352 Fixes: #13352
http://tracker.ceph.com/issues/13375 Fixes: #13375

dachary commented on man/Makefile-server.am:
```diff
@@ -5,8 +5,12 @@ dist_man_MANS += \
-	ceph-debugpack.8 \
+	ceph-debugpack.8
+
+if WITH_SELINUX
```
verified that ceph_selinux.8 is only included with %if 0%{with selinux} in ceph.spec.in, therefore conditionally building it is not going to cause a failure because the package cannot find it.

smithfarm commented:
`suse-wip-infernalis-obs` passes RPM gitbuilders.

smithfarm commented:
```
 /usr/bin/mkdir -p '/srv/autobuild-ceph/gitbuilder.git/build/rpmbuild/BUILDROOT/ceph-9.0.3-2023.g4e643b0.el7.x86_64/usr/share/man/man8'
/usr/bin/install -c -m 644 ceph-syn.8 ceph-conf.8 ceph.8 ceph-authtool.8 rados.8 rbd.8 ceph-post-file.8 ceph-dencoder.8 librados-config.8 ceph-rbdnamer.8 rbd-replay.8 rbd-replay-many.8 rbd-replay-prep.8 cephfs.8 rbd-fuse.8 ceph-fuse.8 radosgw.8 radosgw-admin.8 ceph-deploy.8 crushtool.8 ceph-run.8 mount.ceph.8 ceph-create-keys.8 ceph-rest-api.8 ceph-debugpack.8 ./ceph_selinux.8 ceph-mon.8 monmaptool.8 ceph-clsinfo.8 ceph-detect-init.8 ceph-disk.8 ceph-osd.8 osdmaptool.8 ceph-mds.8 '/srv/autobuild-ceph/gitbuilder.git/build/rpmbuild/BUILDROOT/ceph-9.0.3-2023.g4e643b0.el7.x86_64/usr/share/man/man8'
/usr/bin/install: cannot stat './ceph_selinux.8': No such file or directory 
```

smithfarm commented:
@branto1 Thanks for the review! I addressed your comments. Still can't figure out why the gitbuilder doesn't like the `WITH_SELINUX` conditional in `man/Makefile-server.am`, though - any ideas?

smithfarm commented:
> You were right that ceph_selinux.8 is not in EXTRA_DIST and hence, it is not being 
> packaged properly (if you use make dist). There are some other issues with the man 
> page, though. Can you stash this chunk? I'll create a PR that will hopefully fix that 
> behaviour alongside some other issues.

OK, I removed that commit from the PR.

branto1 commented:
@smithfarm: You did not have to remove the whole commit, just the chunk. Nevermind, I'll use your commit in the other PR. I've scheduled a koji build and if that passes I'll merge this.

smithfarm commented:
@branto1 Thanks. You're welcome to use the commit; it's f48bc75767dd245b631d8b0ce7fa6375dcc496d0

branto1 commented:
The packages built just fine (*). I'll go ahead and merge this.

(*) http://koji.fedoraproject.org/koji/taskinfo?taskID=11350041

branto1 commented:
The new PR: https://github.com/ceph/ceph/pull/6183
